tests/baselines/reference/autoAccessor5(target=es2015).js

```diff
@@ -0,0 +1,53 @@
+//// [autoAccessor5.ts]
+class C1 {
+    accessor ["w"]: any;
+    accessor ["x"] = 1;
+    static accessor ["y"]: any;
+    static accessor ["z"] = 2;
+}
+
+declare var f: any;
+class C2 {
+    // @ts-ignore
```

Comment:
what is the error? Do we not baseline emit when there are errors? I thought we did, but maybe I was thinking of types/symbols.

Reply:
The error is that `f()` isn't a literal or unique symbol type. I can look at changing the test.